### PR TITLE
Add crossnet interface

### PR DESCRIPTION
**Pull request by cryptoAtwill**
Update subnet manager interface with crossnet functions:
- fund
- release
- propagate
- whitelist_propagator

### CODE REVIEW

File: src/manager/lotus.rs [hunk 1]
```diff
@@ -145,6 +145,27 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
...
+        todo!()
+    }
+
+    async fn propagate(&self, _subnet: SubnetID, _from: Address, _postbox_cid: Cid) -> Result<()> {
+        todo!()
+    }
+
+    async fn whitelist_propagator(
+        &self,
+        _subnet: SubnetID,
```
Comment (adlrocha) [inline src/manager/lotus.rs:163]: We need to specify the cid of the message to add the propagator to, right?

File: src/manager/lotus.rs [hunk 2]
```diff
@@ -145,6 +145,27 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
...
+
+    async fn fund(&self, _subnet: SubnetID, _from: Address, _amount: TokenAmount) -> Result<()> {
+        todo!()
+    }
+
+    async fn release(&self, _subnet: SubnetID, _from: Address) -> Result<()> {
+        todo!()
+    }
+
+    async fn propagate(&self, _subnet: SubnetID, _from: Address, _postbox_cid: Cid) -> Result<()> {
```
Comment (adlrocha) [inline src/manager/lotus.rs:157]:
```suggestion
    async fn propagate(&self, _subnet: SubnetID, _from: Address, _postbox_msg_cid: Cid) -> Result<()> {
```

**Review by adlrocha — APPROVED** [review]
LGTM, be sure to change the issues I raised before merging. But apart from that let's keep moving :) Thanks.

File: src/manager/lotus.rs (hunk 1 above)
Comment (cryptoAtwill) [reply src/manager/lotus.rs:168]: updated and resolving this.

**Pull request merged**